src/main/java/client/Client.java

```diff
@@ -427,7 +429,8 @@ public static void main(String args[]) {
 
                      // Otherwise, receive what text
                      else {
-                        appendToChatBox(s + "\n");
+                    	String decryptedContent = encrypter.encrypt(s);
```

Comment:
je decrypt met de functie encrypt? klopt dit?

Reply:
Ik snap m nu ik de encrypter class heb gezien. Maar het leest wel raar haha.

Reply:
Miss niet verkeerd om daar expliciet te zijn of het niet perse encrypt te noemen.

Reply:
wist niet echt wat een passender naam kon zijn dus had het encrypt gelaten :P